### ISSUE TITLE
Remove temporary fix for ros bags once possible

### ISSUE THREAD

**Issue by phinik** [bit-bots/bitbots_vision#352]
## Is your feature request related to a problem? Please describe.
Just a reminder

## Describe the solution you'd like
Remove temporary fix for ros bags once possible (https://github.com/bit-bots/bitbots_vision/pull/343)

**Comment by Flova** [comment]
Good point. As this is a Middleware issue we might want to link the upstream issue. Do you know which one it is? @timonegk @jaagut

**Comment by jaagut** [comment]
https://github.com/ros2/rmw_cyclonedds/issues/461

**Comment by Flova** [comment]
https://github.com/ros2/rmw_cyclonedds/issues/461

**Comment by Flova** [comment]
A nvm you are faster

**Comment by Flova** [comment]
Seems like we need to test this again and answer
